sdk/search/Azure.Search.Documents/src/Generated/Models/AIStudioModelCatalogName.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Azure.Search.Documents.Indexes.Models
 {
-    /// <summary> The name of the embedding model from the Azure AI Studio Catalog that will be called. </summary>
+    /// <summary> The name of the embedding model from the Azure AI Foundry Catalog that will be called. </summary>
```

Comment:
The issue is that you cannot manually update the files in the `Generated` folder. These files are generated from the Swagger definition, and this description is also coming from the Swagger.

Reply:
🤦 Thank you for pointing that out!